seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/seatunnel/SeaTunnelContainer.java

```diff
@@ -338,9 +338,9 @@ private Container.ExecResult doExecuteJob(String confFile, String jobId, List<St
             //            classLoaderObjectCheck(1);
             return result;
         } else {
-            // Waiting 10s for release thread
+            // Waiting 60s for release thread
             Awaitility.await()
-                    .atMost(30, TimeUnit.SECONDS)
+                    .atMost(120, TimeUnit.SECONDS)
```

Comment:
Hi, why change this?

Reply:
Hi, because I add the follwing code into DatabendIT and have no effect, so got a timeout error after e2e test.
```
  Awaitility.given()
                .ignoreExceptions()
                .await()
                .atMost(360L, TimeUnit.SECONDS)
                .untilAsserted(this::initConnection);
```

Do you have some suggestions?

Reply:
Could you share the error test case link?

Reply:
Hi @Hisoka-X , has fixed this problem and change to 30.